packages/valdueza-storybook/.storybook/config.js

```diff
@@ -20,17 +20,20 @@ addParameters({
 
 const req = requireContext('../../../packages', true, /\.stories\.((js|ts)x?)$/);
 
-// This optional decorator removes the material UI style IDs for deterministic snapshots. See:
-// https://github.com/mui-org/material-ui/issues/9492
-// https://github.com/abrcdf1023/egroup-material/commit/a876a4b411fda490a53ee20179acc72fe6514aaa
-addDecorator(story => {
-  const generateClassName = (rule, styleSheet) => `${styleSheet.options.classNamePrefix}-${rule.key}`;
-  return (
-    <StylesProvider generateClassName={generateClassName}>
-      {story()}
-    </StylesProvider>
-  );
-});
+// We need to add decorator for testing only, otherwise we will break visual control of the storybooks
```

Comment:
👍

Reply:
This is now fixed on master: https://github.com/iov-one/ponferrada/pull/491 Thanks for the patch!